WalletWasabi.Tests/Helpers/WabiSabiFactory.cs

```diff
@@ -47,10 +47,12 @@ public static OwnershipIdentifier GetOwnershipIdentifier(Script scriptPubKey)
 
 	public static Round CreateRound(WabiSabiConfig cfg)
 	{
+		using var insecureRandom = new InsecureRandom();
```

Comment:
No, this is not correct because what you basically do is:

1. Create an `InsecureRandom` object.
2. Create `Round` object instance with the `InsecureRandom` instance.
3. Dispose the `InsecureRandom`

-> So `round` now contains disposed `InsecureRandom` instance.